packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -1103,9 +1100,22 @@ function build(
     assert(key.match(/^[-\w.+/]+$/), X`invalid vatstore key`);
   }
 
-  function setBuildRootObject(buildRootObject) {
-    assert(!didRoot);
-    didRoot = true;
+  async function startVat() {
```

Comment:
Let's rearrange this to build `vpow` at the beginning, while we're still in the unmetered turn, so all of our preparations are ready before we call `buildVatNamespace`. I'd like to have a comment that clearly marks the point at which user-provided code might cause us to crash or overrun a meter, and do as much as possible before that point.